Documentation/go_client.md

```diff
@@ -0,0 +1,18 @@
+# etcd go client API
+
+This document describes tips for using the etcd go client (the package `github.com/coreos/etcd/client`).
```

Comment:
This document describes using the etcd client Go package `github.com/coreos/etcd/client`.